src/components/RightSidebar/Matterbridge/MatterbridgeSettings.vue

```diff
@@ -565,38 +602,168 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-.loading {
-	margin-top: 30px;
+::v-deep .icon-slack {
+	mask: url('./../../../../img/bridge-services/slack.svg') no-repeat;
```

Comment:
what magic is `mask` can't we use it just as a normal `img` or as a background image, like with all other icons?

Reply:
It's an easy way to support the dark theme. Other icons have their background-image changed with global CSS variables. While we could also declare CSS variables and make them change depending on dark theme, I find it more convenient to use masks and `background-color: var(--color-main-text)`. As you wish.

Other advantage of using masks, it gets rid of the image color.

Reply:
https://caniuse.com/?search=mask

It's not supported in all necessary browsers. So let's just use the current guidelines. make the svgs black and use the normal way to make it automatically adjust to dark theme.

Reply:
Right. Icons are now `<img>` using black svg images. `theme--dark` body class is now used to apply a CSS filter to those images. [CSS filter support](https://caniuse.com/css-filters).